### ISSUE TITLE
Move GLSL shader code out of deferred example source

### ISSUE THREAD

**Issue by sectopod** [gfx-rs/gfx#895]
We don't want to pollute the generic source files with platform-dependent code. Same may apply to other examples. Please wait for #861 before starting.
Edit: it would be even better to dynamically load the code from the disk. It would allow changing shaders without rebuilding.

**Comment by Binero** [comment]
Cargo seems to rebuild anyway, even if no actual Rust files are updated.

**Comment by sectopod** [comment]
@Binero it depends on how exactly we'll be loading the shaders. If via `include_bin!`, then yes, rebuild is imminent. If via IO, then we can change the shaders at any time.
